cmd/acraserver/listener.go

```diff
@@ -153,26 +310,77 @@ func (server *SServer) handleCommandsConnection(connection net.Conn) {
 
 // start listening commands connections from proxy
 func (server *SServer) StartCommands() {
-	listener, err := network.Listen(server.config.GetAcraAPIConnectionString())
+	var connection net.Conn
+	var listener, err = network.Listen(server.config.GetAcraAPIConnectionString())
 	if err != nil {
-		log.WithError(err).Errorln("can't start listen command connections")
+		log.WithError(err).Errorln("can't start listen command api connections")
+		ErrorSignalChannel <- syscall.SIGTERM
```

Comment:
same about global variable and channel
and to not repeat, will be good to check/apply fixes from `Start` method in this method

Reply:
done